tests/store/artifact/test_databricks_artifact_repo.py

```diff
@@ -429,46 +414,26 @@ def test_log_artifact_with_relative_path(self, test_file, artifact_path, expecte
         with mock.patch(
             DATABRICKS_ARTIFACT_REPOSITORY + "._get_run_artifact_root"
         ) as get_run_artifact_root_mock, mock.patch(
-            DATABRICKS_ARTIFACT_REPOSITORY + "._get_write_credentials"
-        ) as write_credentials_mock, mock.patch(
+            DATABRICKS_ARTIFACT_REPOSITORY + "._get_write_credential_infos"
+        ) as write_credential_infos_mock, mock.patch(
             DATABRICKS_ARTIFACT_REPOSITORY + "._upload_to_cloud"
         ) as upload_mock:
             get_run_artifact_root_mock.return_value = MOCK_RUN_ROOT_URI
             databricks_artifact_repo = get_artifact_repository(MOCK_SUBDIR_ROOT_URI)
-            mock_credentials = ArtifactCredentialInfo(
+            mock_credential_info = ArtifactCredentialInfo(
                 signed_uri=MOCK_AZURE_SIGNED_URI, type=ArtifactCredentialType.AZURE_SAS_URI
             )
-            write_credentials_response_proto = GetCredentialsForWrite.Response(
-                credentials=mock_credentials
-            )
-            write_credentials_mock.return_value = write_credentials_response_proto
+            write_credential_infos_mock.return_value = [mock_credential_info]
             upload_mock.return_value = None
             databricks_artifact_repo.log_artifact(test_file.strpath, artifact_path)
-            write_credentials_mock.assert_called_with(MOCK_RUN_ID, expected_location)
+            write_credential_infos_mock.assert_called_with(
+                run_id=MOCK_RUN_ID, paths=[expected_location]
+            )
             upload_mock.assert_called_with(
-                write_credentials_response_proto, test_file.strpath, expected_location
-            )
-
-    @pytest.mark.parametrize("artifact_path", [None, "output/", ""])
-    def test_log_artifacts(self, databricks_artifact_repo, test_dir, artifact_path):
```

Comment:
This test case previously depended on the assumption that `log_artifacts()` was implemented by repeatedly calling `log_artifact()`, which is no longer the case due to our use of batched credential generation. This test case has been obviated by `test_artifact_logging`.